### ISSUE TITLE
WP 5.9 issues

### ISSUE THREAD

**Issue by dingo-d** [infinum/eightshift-frontend-libs#548]
### Describe your bug

I have found some small issues on new WP version and our libs, mostly UI/UX issues.

### Range slider changes

<img width="307" alt="image" src="https://user-images.githubusercontent.com/8638515/151153370-47d7f3b4-ce49-4cf2-8e7c-88e9aef65490.png">

<img width="261" alt="image" src="https://user-images.githubusercontent.com/8638515/151153680-6d4d1eba-a2fb-412b-ab6c-1ed396bfe04a.png">

The bottom labels are placed a bit weird

### Column appender

<img width="1671" alt="image (2)" src="https://user-images.githubusercontent.com/8638515/151153506-fbcb9095-5171-4aad-a1b0-b5fa61124c3d.png">

Super visible, so that's a plus? XD

### Color picker

<img width="211" alt="image" src="https://user-images.githubusercontent.com/8638515/151170522-3c7aa868-7b7c-4617-94d2-a36e5a0966e4.png">

Reset looks too close to colors

### Media button not working in editor

Not sure if this is specific to my project, but the media button didn't work on editor

<img width="1843" alt="image" src="https://user-images.githubusercontent.com/8638515/151170797-ed052540-a68c-470d-b839-4a27731f2a04.png">

The image was an inner block, so it could be only on inner blocks

### Steps to Reproduce

1. Use latest libs 
2. Use latest WP (5.9)
3. Test to confirm

### Expected behavior

Should be working

### Environment info

- PHP: 7.4
- WSL (Ubuntu 20.04)
- Nginx
- Windows 11
- Libs version: 5.0.2

### Please confirm that you have searched existing issues in this repo.

Yes

### Please confirm that you have searched in our documentation and couldn't find the answer.

Yes

### Please confirm that your bug occurs with all plugins uninstalled and with the default WordPress theme active.

Yes

**Comment by moonbyt3** [comment]
@iruzevic 
I can't see latest UX changes when i setup theme
What I did: 
1. Cloned new boilerplate, with npx, did wp activate theme, everything is working well
2. Pulled eightshift-frontend-libs from repo in separate folder
3. Copy pasted files from repo to @eightshift/frontend-libs
4. Executed npm run start inside theme

I can't see slider labels in the sidebar for any component that uses it
![image](https://user-images.githubusercontent.com/24306614/151821298-344fe219-3ac1-47ae-a005-e9c59f8816f4.png)

Am I missing something?

**Comment by iruzevic** [comment]
yea something like that and this huge plus signs and etc

**Comment by mbmjertan** [comment]
I found some issues with the Carousel block as well:

- this chonky boy:
![Screenshot 2022-03-02 at 14 10 34](https://user-images.githubusercontent.com/1742806/156367891-865fedb7-17ca-43e7-8ecd-4f9e6c767345.png)

- `MediaPlaceholder` upload buttons aren't clickable (because of the block appender being only visually hidden, and covering them)

**Comment by dingo-d** [comment]
Chonky boy 😂 

I solved a lot by placing

```scss
.block-list-appender.wp-block {
	position: relative;
}```

In the `Blocks/assets/styles/editor/_editor.scss` file

**Comment by goranalkovic-infinum** [comment]
> Range slider changes
This was a bug in Gutenberg itself. Looks like it's fixed now (I'm on WP 5.9.2)
![image](https://user-images.githubusercontent.com/77000136/159277264-e19fa942-483d-471f-864e-6404e3f0fa1b.png)

> Color picker
Was rewritten in the meantime 😄 

> Media button not working in editor
Confirmed works fine, with _Image_ by itself and inside Column.

> Column appender
Fixed and tracked in https://github.com/infinum/eightshift-frontend-libs/pull/551#pullrequestreview-915809361

Closing this issue.
